### DISCUSSION TITLE
skywalking-ui   General-Service/Services page loss queryInfo when click next page

### DISCUSSION THREAD

**Discussion by feixiangshenhua** [apache/skywalking#12533] [General]
General-Service/Services page always loss queryInfo where click next page，the first page is right,but when I click next page，the result is wrong(it's loss the query condition). How can solve this problem,any one can help me? thanks so much.
for example,first page,the result is right.
![image](https://github.com/user-attachments/assets/80408436-776a-476d-a973-09575f85379a)
second page,the result is wrong,loss the query condition.
![image](https://github.com/user-attachments/assets/808b78c6-ff08-4f72-9a2e-9f78079c5fce)

**Comment by Fine0830** [comment]
What conditions are missing in your case?

**Reply by feixiangshenhua** [reply]
I solve it，thanks。It's skywalking-ui front code's bug。

**Comment by Fine0830** [comment]
Fixed in https://github.com/apache/skywalking-booster-ui/pull/409
